packages/lms/src/routes/+layout.svelte

```diff
@@ -20,11 +19,28 @@
   const scrollIntoView = (node: HTMLElement) => {
     node.scrollIntoView();
   };
+
+  let isMobile = window.innerWidth <= 768; // Check for mobile viewport width
+  let isNavVisible = false; // Initialize the visibility state
+
+  const handleNavItemClick = () => {
+    if (isMobile) {
+      isNavVisible = !isNavVisible; // Toggle the visibility of the navigation bar
+    }
+  };
+
+  // Add a listener to resize events to toggle the navigation based on viewport width
+  window.addEventListener('resize', () => {
+    isMobile = window.innerWidth <= 768;
+    if (!isMobile) {
+      isNavVisible = false; // Close the navigation on wider screens
+    }
+  });
```

Comment:
I think I wrote about this in his previous PR. We should make use of the available directives in Svelte when possible. It's both going to be more performant and less error-prone down the line. As such, I think we in this case should be utilizing the `<svelte:window />` element available to us instead of adding listeners `onMount` or by importing the `browser` directive.

```svelte
<svelte:window on:resize={handleWindowResize} />`
```